tests/app/helpers/test_template_helpers.py

```diff
@@ -268,30 +329,6 @@ def test_get_page_header_context_census_nisra(app: Flask):
                 "target": "_blank",
             },
         ),
-        (
```

Comment:
`contact-us` link generation has changed and I didn't see any value in updating/adding census tests since we don't use it anymore.

Reply:
Highlights that our current approach won't supported translated link, if they were to have diff link for diff languages. One to keep in mind